README.md

```diff
@@ -46,6 +46,9 @@ Adaptive course authoring platform.
   enter configuration details.
 - Initialize database by running `npm run db migrate`
 - Enable default repository schema by setting env variable `ENABLE_DEFAULT_SCHEMA=1` or define a custom one (more details below).
+- Configure Tailor by following the steps [in the configuration section](#content-repository-structure),
```

Comment:
It feels like this line is a bit overlapping with the previous one 🤔  Consider removing that one and just having this one. 'Configure Tailor....' might be a bit too broad definition / might be worth accenting what is being configured e.g. configure Tailor schema by following.... Before guiding users towards more complex documentation it might be worth having something like: 'To enable basic demo configuration copy the tailor.config.js.example' into..... and then saying something like; for more details on how to do a custom configuration.....
Thoughts?

Reply:
Agreed!